### PR TITLE
homework DB week2

### PR DESCRIPTION
**Pull request by myshuker**

### CODE REVIEW

File: databases/week2/Part2.sql [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+begins_date DATETIME NOT NULL, 
+ends_date DATETIME NOT NULL
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- Student: with the columns: id, name, email, phone, class_id (foreign key)
+
+create table Student(
+ id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+ name varchar(255) NOT NULL, 
+ email varchar(255) NOT NULL,
```
Comment (moonflare) [inline databases/week2/Part2.sql]: It's good practice to have `UNIQUE` for the email column 🙌

File: databases/week2/Part2.sql [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+create database School_database;
+
+-- use database :
+ use School_database; 
+ 
+ -- create tables:
+-- Class: with the columns: id, name, begins (date), ends (date) 
+
+
+ create table Class(
```
Comment (moonflare) [inline databases/week2/Part2.sql]: It's good practice to have `database names` and `table names` in lower case. It avoids differences on case sensitivity between UNIX/Windows 🙌

File: databases/week2/Part2.sql [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+create table Student(
+ id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+ name varchar(255) NOT NULL, 
+ email varchar(255) NOT NULL, 
+ phone varchar(255) NOT NULL, 
+ class_id int(10) unsigned NOT NULL,
+ CONSTRAINT `fk_class` FOREIGN KEY (`class_id`) REFERENCES `Class`(`id`) ON DELETE CASCADE ON UPDATE CASCADE
+)ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+-- Create an index on the name column of the student table.
```
Comment (moonflare) [inline databases/week2/Part2.sql]: Did you miss this one?

File: databases/week2/Part2.sql [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+-- add student 
+insert into Student (id, name, email, phone, class_id ) values (1, 'pero soso', 'pero@gmail.com', '483-396-8795',1);
+insert into Student (id, name, email, phone, class_id) values (2, 'dado isis', 'dado@gmail.com', '635-572-8467', 2);
+insert into Student (id, name, email, phone, class_id) values (3, 'Pablo Koso', 'palo@gmail.com', '790-962-8683', 1);
+insert into Student (id, name, email, phone, class_id) values (4, 'Mero Soro', 'mero@gmail.com', '646-743-6191', 3);
+insert into Student (id, name, email, phone, class_id) values (5, 'Horo Foro', 'heo@gmail.com', '302-678-7931', 3);
+insert into Student (id, name, email, phone, class_id) values (6, 'gogo were', 'gogo@gmail.com', '251-524-6594', 2);
+
+
+-- Add a new column to the class table named status which can only have the following values: not-started, ongoing, finished (hint: enumerations).
```
Comment (moonflare) [inline databases/week2/Part2.sql]: Please read the exercise again. You don't need a new table. Also, you need to use `ENUMS`

File: databases/week2/Part4.sql [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+-- crate tables: 
+
+--create meal table:
+CREATE TABLE `meal` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `title` varchar(255) NOT NULL,
+  `description` text NULL DEFAULT NULL,
+  `location` varchar(255) NOT NULL,
+  `meal_when` DATETIME NOT NULL,
+  `max_recervation` int(10) unsigned NOT NULL,
```
Comment (moonflare) [inline databases/week2/Part4.sql]:
```suggestion
  `max_reservations` int(10) unsigned NOT NULL,
```

File: databases/week2/Part2.sql (hunk 1 above)
Comment (myshuker) [reply databases/week2/Part2.sql]: Sorry Dan , I dont understand what you mean exactly ... maby you can explain that for me tomorrow ....

File: databases/week2/Part2.sql (hunk 3 above)
Comment (myshuker) [reply databases/week2/Part2.sql]: ?

File: databases/week2/Part2.sql (hunk 4 above)
Comment (myshuker) [reply databases/week2/Part2.sql]: alter table class
add status ENUM('not-started', 'ongoing', 'finished') NOT NULL;

File: databases/week2/Part2.sql (hunk 1 above)
Comment (myshuker) [reply databases/week2/Part2.sql]: Are you mean like this ?:
 email varchar(255) unique NOT NULL ,

File: databases/week2/Part2.sql (hunk 3 above)
Comment (myshuker) [reply databases/week2/Part2.sql]: sorry ... what I missed?

File: databases/week2/Part2.sql (hunk 1 above)
Comment (babakbandpey) [reply databases/week2/Part2.sql]: @myshuker Yes that is it.

File: databases/week2/Part2.sql (hunk 3 above)
Comment (babakbandpey) [reply databases/week2/Part2.sql]: The answer to this one :)
-- Create an index on the name column of the student table.

File: databases/week2/Part2.sql (hunk 1 above)
Comment (myshuker) [reply databases/week2/Part2.sql]: ok thanks

File: databases/week2/Part2.sql (hunk 3 above)
Comment (myshuker) [reply databases/week2/Part2.sql]: should I use this syntax right?
CREATE INDEX index_name ON table_name (column_list)

Comment (myshuker) [reply databases/week2/Part2.sql]: CREATE INDEX index_name ON Student (name);

Comment (babakbandpey) [reply databases/week2/Part2.sql]: https://dev.mysql.com/doc/refman/5.7/en/create-index.html

```
CREATE [UNIQUE | FULLTEXT | SPATIAL] INDEX index_name
    [index_type]
    ON tbl_name (key_part,...)
    [index_option]
    [algorithm_option | lock_option]
```

You can run that and see what happens anyway.

**Pull request closed without merging**